spec/models/builder_spec.rb

```diff
@@ -173,37 +184,16 @@ def file_contains(path, fragment)
       #   'spec.add_dependency "rails-assets-desandro--get-size", ">= 1.1.4", "< 2.0"'
     end
 
-    # This is strange example because colorbox has only
-    # colorbox.css files that are examples (about 5 of them)
-    # Rails Assets algorithm select first example colorbox.css
-    # because it is named the same way as the gem
-    #
-    # I'm not sure it is bug of feature, so I'm leaving it :-)
-    component "colorbox", "1.5.5" do
-      gem_file "app/assets/stylesheets/colorbox.scss"
-      gem_file "app/assets/stylesheets/colorbox/colorbox.scss"
-      file_contains "app/assets/stylesheets/colorbox.scss",
-        "@import 'colorbox/colorbox.scss';"
-    end
-
-    # main is hash
-    component "orbicular", "1.0.3" do
-      gem_file "app/assets/stylesheets/orbicular.scss"
-      gem_file "app/assets/stylesheets/orbicular/orbicular.scss"
-      gem_file "app/assets/javascripts/orbicular.js"
-      gem_file "app/assets/javascripts/orbicular/orbicular.js"
-    end
-
     # ^1.2.3 kind of dependency
     component "jquery-masonry", "3.1.5" do
-      gem_file "app/assets/javascripts/jquery-masonry.js"
```

Comment:
Frankly I don't understand why these changed. The pattern here is that these components are registered by the name `jquery-masonry` but bower used to (and still does) say that its name is just `masonry`. Very confusing. When you do `bower info jquery-masonry#1.3.5` it says its `name` is `masonry`. It does that in both the old and new versions of `bower`.